ui-admin/src/study/participants/participantList/facets/IntRangeFacetView.tsx

```diff
@@ -0,0 +1,35 @@
+import { FacetValue, IntRangeFacetValue } from 'api/enrolleeSearch'
+import React from 'react'
+
+/**
+ *
+ */
+const IntRangeFacetView = ({ facetValue, updateValue }:
+                             {facetValue: IntRangeFacetValue,
+                               updateValue: (facetValue: FacetValue | null) => void}) => {
+  const min = facetValue.min ?? ''
+  const max = facetValue.max ?? ''
+  const textToVal = (text: string): number | null => {
```

Comment:
Since this function doesn't depend on anything within the component, suggest moving it out to the module level to avoid recreating it on every render.